Libraries/LibGUI/ComboBox.cpp

```diff
@@ -186,8 +186,13 @@ void ComboBox::open()
         model()->row_count() * m_list_view->item_height() + m_list_view->frame_thickness() * 2
     };
 
+    auto taskbar_height = 28;
+    auto menubar_height = 19;
```

Comment:
Let's not hardcode this here, instead please add some getters to `GUI::Desktop`.
They can return hardcoded values, but at least we'll have them in a logical place if we want to make these dynamic in the future :)